core/src/main/java/org/apache/carbondata/scan/result/vector/CarbonColumnarBatch.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.carbondata.scan.result.vector;
+
+public class CarbonColumnarBatch {
```

Comment:
In spark the name is aready `ColumnarBatch` so I just used this name to avoid confusion